bindings/rust-examples/session-resumption/certs/generate.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# immediately bail if any command fails
+set -e
+
+echo "generating self-signed certificate"
```

Comment:
Instead of generating new certificates, should we move the "certs" folder from the `client-hello-resolver` example into the `rust-examples` directory, and then have other examples depend on that?